InertiaCore/Utils/Vite.cs

```diff
@@ -0,0 +1,147 @@
+using System.Text.Encodings.Web;
+using System.Text.Json;
+using Microsoft.AspNetCore.Html;
+using Microsoft.AspNetCore.Mvc.Rendering;
+
+namespace InertiaCore.Utils;
+
+public static class Vite
+{
+
+    private static string GetString(IHtmlContent content)
+    {
+        var writer = new System.IO.StringWriter();
+        content.WriteTo(writer, HtmlEncoder.Default);
+        return writer.ToString();
+    }
+
+    private static string publicDir(string path)
+    {
+        return "wwwroot/build/" + path;
+    }
```

Comment:
There is currently no way to change this path. In `Vite.php` it's called `$buildDirectory` and it's stored as a protected attribute, which can be changed using methods.